### PR TITLE
Implement subforms and Toolbox schema

### PR DESCRIPTION
**Pull request by MaxNoe**
This adds the possibility to define subforms, i.e. nesting forms in forms.

This is usefull for creating groups of inputs that belong together, E.g. multiple-choice checkboxes where more than one can be chosen.

### CODE REVIEW

File: populate_database.py [hunk 1]
```diff
@@ -26,25 +32,11 @@
 if Event.query.first() is None:
     print('creating event')
     event = Event(
-        name='Workshop 2019',
+        name='Toolbox Workshop 2019',
```
Comment (chrbeckm) [inline populate_database.py:35]: should be 2020

File: member_database/events/common_schemata.py [hunk 2]
```diff
@@ -66,6 +66,76 @@
...
+                'math': {'type': 'boolean', 'label': 'Formelsatz'},
+                'toc': {'type': 'boolean', 'label': 'Automatisierung von Inhalts- und anderen Verzeichnissen'},
+                'bib': {'type': 'boolean', 'label': 'Korrektes Zitieren und Literaturverzeichniss'},
+                'tikz': {'type': 'boolean', 'label': 'Zeichnungen mit TikZ'},
+                'beamer': {'type': 'boolean', 'label': 'Präsentationen mit Beamer'},
+            },
+        },
+        'other_interesets': {'type': 'string', 'label': 'Mich interessiert außerdem'},
+        'remarks': {'type': 'string', 'label': 'Weitere Anmerkungen (z.B. Terminkollisionen)'},
+    }
```
Comment (chrbeckm) [inline member_database/events/common_schemata.py:145]: possible questions:
- betreutes Installieren (ja/nein)
- usage of zoom/discord

File: populate_database.py (hunk 1 above)
Comment (MaxNoe) [reply populate_database.py:35]: This is just a script for local testing,  nothing to do with the actual deployed website

**Comment by bixel** [comment]
Looks good as far as I can tell. I still cannot Test though.
The only thing i see: There should be a version bump.

**Comment by MaxNoe** [comment]
Version bump is in #98, I'll make a release with these three PRs

**Pull request merged**